### PR TITLE
Add user delete and get by email and filter mail in date range by uid

### PR DESCRIPTION
**Pull request by parsakmt**

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -73,29 +91,39 @@ def post_mail():
         return "Internal Server Error", 500
 
 
+@app.get("/users")
+def get_users():
```
Comment (rileysw) [inline app.py:95]: what is the use case for getting all users?

File: app.py [hunk 2]
```diff
@@ -122,5 +150,17 @@ def post_new_user():
     except Exception as err:
         return f"Internal Server Error", 500
 
+
+@app.route("/users/<email>", methods=["DELETE"])
+def delete_user_email(email):
```
Comment (rileysw) [inline app.py:155]: not a use case right? I'm assuming for testing purposes?

**Review by rileysw — APPROVED** [review]

File: app.py (hunk 1 above)
Comment (parsakmt) [reply app.py:95]: primarily for testing on our end to see the list of users that have been setup

File: app.py (hunk 2 above)
Comment (parsakmt) [reply app.py:155]: for testing purposes but also probably will have a delete account option in the app so we don't have to make a delete request everytime

**Pull request merged**